### ISSUE TITLE
Plugin is expected a map variable in context

### ISSUE THREAD

**Issue by tomacla** [Raruto/leaflet-elevation#181]
### Subject of the issue

Plugin not working when map object variable is not called explicitly "map".

### Your environment
* leaflet-elevation: 1.9.2
* leaflet: 1.7.1
* browser: chrome
* operating system: linux ubuntu
* link to your project: https://gpx.n-peloton.fr/

### Steps to reproduce

If I initialize leaflet map like this : 

```js
var map = L.map(mapContainerId).setView([51.505, -0.09], 13);
```

Everything work fine. But if I write

```js
var target = L.map(mapContainerId).setView([51.505, -0.09], 13);
```

There is a failure due to the fact that lealet-elevation is referencing a map variable in the code that is that necessery existing.
Like the following line for example : 

```js
 this._container = d3.select(map.getPane('elevationPane')).select("svg > g").attr("class", "height-focus-group");
```

### Expected behaviour

Leaflet-elevation should work whatever the name of the map variable.

### Actual behaviour

Leaflet-elevation fail if the map variable is not called "map"

**Comment by mondinnic** [comment]
I have the same problem, with 1.7.5 all works well!

**Comment by Raruto** [comment]
Yep, there has been some house cleaning in the latest releases ([4695e6b](https://github.com/Raruto/leaflet-elevation/commit/4695e6b3c17fa365750d07a8fcc0ed738dd1a814#diff-c9adbd8ac5b16569b59c38a363bb3f4d4d9d3af675c4e26b25775f4aaafd0f5c)), anyway, version **1.9.3** should solve this.

_Happy mondays,
Raruto_
